de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -80,6 +84,11 @@ protected void finished(Description description) {
 	protected static final String PROJECTNAME = "SWTBotTestProject";
 	protected IProject project;
 	
+	protected static final int MAX_TEST_CASE_TIMEOUT_SECONDS = 90;
+	
+	@Rule
+    public Timeout globalTimeout = Timeout.seconds(MAX_TEST_CASE_TIMEOUT_SECONDS);
```

Comment:
Weird whitespacing.

Reply:
Indeting was mixed by ws and tab. Fixed it.